### PR TITLE
🌦️ Environment Changes

### PR DESCRIPTION
**Pull request by SatanshuMishra**
- Installed Tauri into the project.
- Added basic Tauri Config.

### CODE REVIEW

File: .changeset/healthy-stingrays-laugh.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"CulinaryCache": minor
+---
+
+Installed and Configured Tauri for the Project
```
Comment (VoxelMC) [inline .changeset/healthy-stingrays-laugh.md:5]:
```suggestion
Installs and Configures Tauri for the Project
```
Generally good to choose a syntax for the changesets.

Astro uses grammar such that it completes the sentence: "This PR"

So, 
Installs and configures Tauro for the project
satisfies
`This PR`  Installs and configures Tauro for the project

Obviously this is not the only way to do it, and is just one way, but it's good to set it now and get used to it sooner than later!

**Review by VoxelMC — APPROVED** [review]
LGTM! One comment to ruminate over, but merge when ready!

File: .changeset/healthy-stingrays-laugh.md (hunk 1 above)
Comment (SatanshuMishra) [reply .changeset/healthy-stingrays-laugh.md:5]: Agreed. I do not mind doing it similar to how Astro does it. But as you mentioned, we shouldn't limit ourselves to the same format. Individually, lets each come up with a list of things we want to be mentioned in a `changeset` summary. We can build a standard to follow from that. Lets discuss this tonight.

Comment (SatanshuMishra) [reply .changeset/healthy-stingrays-laugh.md:5]: Reference #7 to organize our thoughts regarding the above recommendation.

**Pull request merged**